### PR TITLE
Implementing rumble

### PR DESCRIPTION
**Pull request by Iliana0117**
## Overview

This PR will... create sequential command from intake note with new commands + implement rumble for when we have intaked a note.

## Tests Ran
 builds

### CODE REVIEW

**Review by Ash1R — APPROVED** [review]

**Review by Pennedrose — COMMENTED** [review]
The structure of intakeNote has changed during testing, to now something that works. This code will be merged into main tonight, and then could this PR be changed to reflect those changes? Then it will be merged

**Comment by Ash1R** [comment]
Is this ready for merge?

**Comment by omegaladon** [comment]
is this still relevant?

File: src/main/java/frc/robot/RobotContainer.java [hunk 1]
```diff
@@ -67,8 +67,22 @@ public RobotContainer(RobotId robotId) {
...
 
       case TestBed1:
+        // GameController kDriver = new GameController(Constants.DRIVER_JOY);
+        // kDriver.get(Button.X).onTrue(new InstantCommand(() -> kDriver.setRumble(RumbleStatus.RUMBLE_ON)));
+        // kDriver.get(Button.X).onFalse(new InstantCommand(() -> kDriver.setRumble(RumbleStatus.RUMBLE_OFF)));
+        intake = new Intake();
         index = new StorageIndex();
-        shooter = new Shooter();
+        operator = new Operator(intake, index);
+        operator.configureControls((x)->{
```
Comment (omegaladon) [inline src/main/java/frc/robot/RobotContainer.java:76]: this should not be in robot container. move it to the operator class

Comment (omegaladon) [reply src/main/java/frc/robot/RobotContainer.java:76]: oh, and also please rename all instances of variables named "x" to something more explanatory, like hasNote

File: src/main/java/frc/robot/RobotContainer.java [hunk 2]
```diff
@@ -98,11 +112,20 @@ public RobotContainer(RobotId robotId) {
         //SignalLogger.start();
 
         driver.configureControls();
-        operator.configureControls();
+        operator.configureControls((x)->{
```
Comment (omegaladon) [inline src/main/java/frc/robot/RobotContainer.java:115]: same as above comment

File: src/main/java/frc/robot/commands/gpm/IntakeNote.java [hunk 3]
```diff
@@ -1,26 +1,54 @@
...
+
+        addRequirements(intake, storageIndex, arm);
+        // addRequirements(intake, storageIndex);
+
+    }
     public IntakeNote(Intake intake, StorageIndex storageIndex, Arm arm) {
         this.intake = intake;
         this.storageIndex = storageIndex;
         this.arm = arm;
+        this.reactor = (x)->{
```
Comment (omegaladon) [inline src/main/java/frc/robot/commands/gpm/IntakeNote.java:40]: rename "x"

**Comment by Kyle-Eldridge** [comment]
Is this PR ready for review?

**Pull request closed without merging**